packages/wrangler/src/cloudchamber/common.ts

```diff
@@ -260,7 +260,7 @@ async function fillOpenAPIConfiguration(config: Config, json: boolean) {
 			message = JSON.stringify(err);
 		}
 
-		crash("loading Cloudchamber account failed:" + message);
+		crash("loading account failed: " + message);
```

Comment:
I just changed this in the other PR

```suggestion
		crash("Loading account failed: " + message);
```